R/project.R

```diff
@@ -124,7 +129,9 @@
 #'
 
 project <- function(x, R, si, n_sim = 100, n_days = 7,
-                    R_fix_within = FALSE) {
+                    R_fix_within = FALSE,
+                    distr = "poisson",
```

Comment:
rename `distr` to `model` everywhere, and then change argument defaults to 
```
model = c("poisson", "negbin"), 
```
and first thing in the function body:
```
model <- match.arg(model)
```